cms/templates/partials/subnav.html

```diff
@@ -10,7 +10,7 @@
       {% if page.testimonials %} <li class="nav-item px-2 d-flex align-items-center"><a class="nav-link text-center" href="#testimonials">What Learners Are Saying</a></li> {% endif %}
       {% if page.faculty %} <li class="nav-item px-2 d-flex align-items-center"><a class="nav-link text-center" href="#faculty">Your MIT Faculty</a></li> {% endif %}
       {% if page.course_lineup %} <li class="nav-item px-2 d-flex align-items-center"><a class="nav-link text-center" href="#courseLineup">Courses In This Program</a></li> {% endif %}
-      {% if page.for_teams %} <li class="nav-item px-2 d-flex align-items-center"><a class="nav-link text-center" href="#forTeams">For Teams</a></li> {% endif %}
+      {% if page.for_teams %} <li class="nav-item px-2 d-flex align-items-center"><a class="nav-link text-center" href="#forTeams">Enterprise</a></li> {% endif %}
```

Comment:
Do we need to update the `for_teams` to `for_enterprise` too in CMS Wagtail ?

Reply:
@ahmed-arbisoft your thought over it ?

Reply:
@asadiqbal08 @pdpinch when we built this we did not have much info. This looks fine as far as the intended change is concerned, my $0.02 is that we should remove the hard coded text completely and use the titles set on the pages themselves like: `page.for_teams.title` and `page.course_lineup.title` unless you have something else in mind.

Reply:
Good idea! Can we do that? I'll open an issue.

Reply:
The marketing team wants more flexibility for headings than this would allow, so we're just going to leave as is for now.